ui/page/components/votebar_widget.go

```diff
@@ -260,12 +260,13 @@ func (v *VoteBar) layoutIconAndText(gtx C, lbl decredmaterial.Label, count int,
 				return lbl.Layout(gtx)
 			}),
 			layout.Rigid(func(gtx C) D {
-				percentage := (count / v.totalVotes) * 100
+				count := float64(count)
+				percentage := (count / float64(v.totalVotes)) * 100
 				if percentage < 0 {
```

Comment:
Can `count` or `v.totalVotes` be less than 0? If that's not possible, this check is useless.

Reply:
Weird, but okay. I think it'd be good to log if either count or v.totalVotes were negative.

Reply:
@Sirmorrison are we going to be logging the output in this PR?